### DISCUSSION TITLE
Auth and World servers on separate machines?

### DISCUSSION THREAD

**Discussion by Billy-d-Kidd** [azerothcore/azerothcore-wotlk#19181] [General]
Hello all, I'm new here and was reading through some of the installation manuals, and was wondering if it would be possible to run the authserver on one machine and worldserver on a different machine and have them networked together?

**Comment by CenturionMurda** [comment]
I am about to start working on this myself I don't think it would be to bad other than getting the database entries correct so that each realm entry on the authentication server points to the correct realm IP for the wow server.
